components/help-dropdown/help-dropdown.jsx

```diff
@@ -23,7 +23,7 @@ export default class HelpDropdown extends React.Component {
     });
 
     return (
-      <select className="custom-select w-100" value={ this.props.value } onChange={ (event) => {this.props.helpType(event);} }>
+      <select id={this.props.id} className="custom-select w-100" value={ this.props.value } onChange={ (event) => {this.props.helpType(event);} }>
```

Comment:
In case we reuse this component, let me know if it'd be better to add a special class instead of `id`.

Reply:
https://github.com/mozilla/network-pulse/pull/1102/files#r297807293